diff --git a/src/other/gdal/CMakeLists.txt b/src/other/gdal/CMakeLists.txt
index 505b288bb7..38de430d2b 100644
--- a/src/other/gdal/CMakeLists.txt
+++ b/src/other/gdal/CMakeLists.txt
@@ -83,23 +83,8 @@ include(CheckCXXCompilerFlag)
 
 find_package(SSE) # checks SSE, AVX and AVX2
 
-CHECK_SSE(CXX "AVX" " ;-mavx;/arch:AVX")
-IF(C_AVX_FOUND)
-  MESSAGE(STATUS "AVX Found")
-  add_definitions(-DHAVE_AVX_AT_COMPILE_TIME)
-ENDIF(C_AVX_FOUND)
-
-CHECK_SSE(CXX "SSE" " ;-msse;/arch:SSE")
-IF(C_SSE_FOUND)
-  MESSAGE(STATUS "SSE Found")
-  add_definitions(-DHAVE_SSE_AT_COMPILE_TIME)
-ENDIF(C_SSE_FOUND)
-
-CHECK_SSE(CXX "SSSE3" " ;-mssse3;/arch:SSE3")
-IF(C_SSSE3_FOUND)
-  MESSAGE(STATUS "MSSSE3 Found")
-  add_definitions(-DHAVE_SSSE3_AT_COMPILE_TIME)
-ENDIF(C_SSSE3_FOUND)
+add_definitions(-DHAVE_SSE_AT_COMPILE_TIME)
+add_definitions(-DHAVE_SSSE3_AT_COMPILE_TIME)
 
 add_definitions(-DPROJ_STATIC)
 add_definitions(-DGDAL_COMPILATION)
